DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -784,9 +784,7 @@ internal DiscordThreadChannel InternalGetCachedThread(ulong threadId)
 
         internal DiscordChannel InternalGetCachedChannel(ulong channelId)
         {
-            DiscordDmChannel foundDmChannel = default;
-
-            if (this._privateChannels?.TryGetValue(channelId, out foundDmChannel) == true)
+            if (this._privateChannels?.TryGetValue(channelId, out var foundDmChannel) == true)
```

Comment:
this isn't JS; why are we doing `== true`

Reply:
![image](https://user-images.githubusercontent.com/784448/133934766-dd53287f-589b-4942-bf43-825e54471e0c.png)

Reply:
The alternative to `left == true` is `left ?? false`